tests/unit_tests/gebt_poc/test_solver.cpp

```diff
@@ -625,7 +625,7 @@ TEST(SolverTest, ConstraintsResidualVector) {
     ConstraintsResidualVector(generalized_coords, position_vectors, constraints_residual);
 
     openturbine::gen_alpha_solver::tests::expect_kokkos_view_1D_equal(
-        constraints_residual, {0.1, 0., 0.12, 0.1, -0.01198, 0.1194}
+        constraints_residual, {0.1, 0., 0.12, 0.1, 0., 0.0}
```

Comment:
Let's consider adding a comment that we expect the `constraints_residual` to be exactly same as the `generalized_coords` where the rotation part went through a log map.


```
// constraints_residual should be same as the generalized_coords where
// q{0.9987502603949662, 0.049979169270678324, 0., 0.} -> v{0.1, 0., 0.}
constraints_residual, {0.1, 0., 0.12, 0.1, 0., 0.}
```

Reply:
Added